package.json

```diff
@@ -71,7 +71,8 @@
     "docs:cp_defs": "mkdir docs/docs_generate; cp source/danger.d.ts docs/docs_generate; cp node_modules/@octokit/rest/index.d.ts docs/docs_generate/github.d.ts",
     "docs": "yarn run docs:cp_defs; yarn typedoc --ignoreCompilerErrors --mode modules --json docs/js_ref_dsl_docs.json --includeDeclarations source",
     "dts-lint": "yarn run declarations && yarn dtslint types",
-    "danger:prepush": "yarn build:fast; yarn test:fixtures; node distribution/commands/danger.js local --base main --dangerfile dangerfile.lite.ts"
+    "danger:local": "node distribution/commands/danger.js local --base main --dangerfile dangerfile.lite.ts",
+    "danger:prepush": "npm-run-all build:fast test:fixtures danger:local"
```

Comment:
Do we really need this `npm-run-all` dependency? Is it just to have a lighter-weight syntax to run multiple commands?

`yarn --silent build:fast && yarn --silent test:fixtures && yarn --silent danger:local` seems to produce the same result without adding a new dependency, unless I’m missing something?

Reply:
this PR is based on changes from https://github.com/danger/danger-js/pull/1254 branch